includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -364,13 +367,37 @@ protected function get_row_data( $item ) {
 					$course_title = '<strong><a class="row-title" href="' . esc_url( $url ) . '">' . apply_filters( 'the_title', $item->post_title, $item->ID ) . '</a></strong>';
 				}
 
+				// Fetch learners in course.
+				$course_args           = array(
+					'post_id' => $item->ID,
+					'type'    => 'sensei_course_status',
+					'status'  => 'any',
+				);
+				$course_students_count = Sensei_Utils::sensei_check_for_activity( apply_filters( 'sensei_analysis_course_learners', $course_args, $item ) );
+
+				// Fetch course stats completed lessons and total lessons.
+				$stats                   = Sensei()->course->get_progress_stats( $item->ID );
+				$lesson_completions      = $stats['completed_lessons_count'];
+				$total_lessons           = $stats['lessons_count'];
+				$average_course_progress = 0;
+
+				if ( $course_students_count && $total_lessons ) {
+					// Average course progress is calculated based on lessons completed for the course
+					// divided by the total possible lessons completed.
+					$average_course_progress = $lesson_completions / ( $course_students_count * $total_lessons ) * 100;
+					$average_course_progress = esc_html(
+					/* translators: Progress value. */
```

Comment:
Looks like this comment is not needed here?

Reply:
Removed it [here](https://github.com/Automattic/sensei/pull/4917/files#diff-6ce1dc075c4832d5bf050e621bcfecbab91d97504e8bc565f4f476b1ea2f654dR412)